docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+
+services:
+  package:
+    image: gradle:7.4.2-jdk8
```

Comment:
I think it's better to have full control over which java version we will be using. You can use a version like this `azul/zulu-openjdk:8` and then copy the gradle if the image does not provide one to you :)

Reply:
After discussion with @brunoocasali we agreed that we can keep this image because the `gradle:7.4.2-jdk8` came from the official [Gradle image](https://hub.docker.com/_/gradle), who used the offical [eclipse-temurin](https://hub.docker.com/_/eclipse-temurin) and we are able to choose which `jdk` version we want.